py/ngff_zarr/tiff_to_ngff_image.py

```
@@ -429,6 +429,91 @@ def _extract_ome_pixel_metadata(
+        - units: Mapping of dimension names to translation units.
+        Returns (None, None) if OME metadata is not available or cannot be parsed.
+    """
+    if not tif.is_ome or not tif.ome_metadata:
+        return None, None
+
+    try:
+        from tifffile import xml2dict
+        ome_metadata = xml2dict(tif.ome_metadata)
+        if "OME" in ome_metadata:
+            ome_metadata = ome_metadata["OME"]
+    except:
```

Ruff (0.16.1)

[error] Do not use bare `except`

(E722)